### ISSUE TITLE
[Content organization] Content organization and/or categorization isn't clear or actionable.  (02.06.4)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#52087]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
Enrollment verification

#### Point of Contact/Reviewers
Cheryl Evans - @CherylEvans - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content organization and/or categorization isn't clear or actionable. 

### Issue Details
Issue: User 301 > Your monthly enrollments verification > Action link (Verify all enrollments) > “All enrollments”, is not very specific.

### VA.gov Experience Standard
[Category Number 02, Issue Number 06](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1

### Platform Recommendation
Recommendation: Add additional clarifying verbiage, such as, “Verify all new and existing enrollments”

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by btruongafs** [comment]
@TamGIBill confirmed no change needed to verbiage.

**Comment by katherine-fung** [comment]
Hi @btruongafs, 

For the alert, can you verify which of these sentences is correct? 

"We've paused your payments because it's been 2 months since you verified your enrollment." 

OR 

"We've paused your payments because it's been more than 2 months since you verified your enrollment." 

Can you please edit the alert to include whichever sentence is correct (and remove the sentence that contains the phrase "2 months in a row?

Can you please also make the following changes: 

- Remove the link "Verify all enrollments" from the alert 

- Change the text for the enrollment verification link under each month to say "Verify your enrollment" 

Once you make these changes, this ticket should be resolved. Thank you!

**Comment by btruongafs** [comment]
@TamGIBill approves changes

**Comment by btruongafs** [comment]
Content changes have been implemented and deployed to staging. Closing out.
